Sources/ExcelExport/ExcelExport.swift

```diff
@@ -165,14 +172,20 @@ public class ExcelExport {
         }
         
         var sheetsValues = [String]()
+        var remainingSpan = [RemainingSpan]()
         for sheet in sheets {
             
             // build sheet
+            var vIndex = 0
             var rows = [String]()
             for row in sheet.rows {
-
+                
```

Comment:
@huguesf this one - as I said super nit so not worth blocking this...

Reply:
:+1: